internal/controllers/v1controllers/allocation_pools.go

```diff
@@ -1,27 +1,21 @@
-// AllocationPoolController, public interface for AllocationPool objects
-
-// Package allocationPools defines data structure representing
-// a allocationPool instance and methods for interacting with them
-// it is left to concrete implementations in package db or others to
-// implement these interfaces
-package allocationpools
+package v1controllers
```

Comment:
update comment? I'm pushing to have a top-level comment to describe what goes (or shouldn't go) in a file.